admin/worker/worker.go

```diff
@@ -110,9 +112,7 @@ func (w *Worker) scheduleCron(ctx context.Context, name string, fn func(context.
 			return nil
 		case <-time.After(waitDuration):
 			err := w.runJob(ctx, name, fn)
-			if err != nil {
-				return err
```

Comment:
I think it better not to exit the admin process when autoscaler is not available.